pallets/payments/src/tests.rs

```diff
@@ -105,7 +107,7 @@ fn check_balance_cancellation() {
 #[test]
 fn test_pay_and_release_works() {
 	new_test_ext().execute_with(|| {
-		let fees: Fees<Test> = build_payment();
+		let fees: Fees<Test> = build_payment(true);
```

Comment:
Passing booleans as-is to function parameters makes it hard to reason about what the parameter does, consider factoring it out to a `const` :)